src/View/Helper/Navigation/AbstractHelper.php

```diff
@@ -282,7 +284,7 @@ public function findActive(
                 [
                     'authorization' => $this->getUseAuthorization() ? $this->getAuthorization() : null,
```

Comment:
IMO 
```php
protected bool $useAuthorization = true;
```
Should have a default value of false. So that accept can return immediately if an Authorization component has not been set or the flag mutated. Setting an authorization instance should mutate the flag.